quinn-proto/src/connection/streams/mod.rs

```diff
@@ -203,7 +203,16 @@ pub struct SendStream<'a> {
 #[allow(clippy::needless_lifetimes)] // Needed for cfg(fuzzing)
 impl<'a> SendStream<'a> {
     #[cfg(fuzzing)]
-    pub fn new(
+    pub fn new_for_fuzzing(
```

Comment:
bikeshed: I'd prefer that this always be named `new`. We could achieve that by having two definitions, selected by `#[cfg(fuzzing)]` state and differing only in visibility, both of which delegate to a private `new_inner` or similar.